src/projects/apps/authentication.md

```diff
@@ -0,0 +1,24 @@
+# App Authentication & Authorization
+
+Authentication is a way of controlling who can access your app by requiring users to log into your application.
```

Comment:
May be worth linking out to external resource https://auth0.com/docs/get-started/identity-fundamentals/authentication-and-authorization